acceptance/tests/vault/main_test.go

```diff
@@ -1,6 +1,7 @@
 package vault
 
 import (
+	"fmt"
 	"os"
 	"testing"
 
```

Comment:
I can back this one out, but right now the Vault tests are flaky everywhere, we have a ticket to address them.